components/compliance-service/api/jobs/server/server.go

```diff
@@ -89,19 +78,8 @@ func (srv *Server) getComplianceAndSecretsConnection(connectionFactory *secureco
 		return
 	}
 
-	// dial secrets
-	secretsConn, err := connectionFactory.Dial("secrets-service", secretsEndpoint)
-	if err != nil {
-		logrus.Errorf("getComplianceAndSecretsConnection error grpc dialing to secrets service")
-		return
-	}
-	// get secrets client with secrets conn
-	secretsClient := secrets.NewSecretsServiceClient(secretsConn)
-	if secretsClient == nil {
-		logrus.Errorf("getComplianceAndSecretsConnection, could not obtain secrets service client: %s", err)
-		return
-	}
-	srv.schedulerServer = scheduler.New(mgrClient, nodesClient, db, ingestClient, secretsClient, remoteInspecVer)
+	scanner := scanner.New(mgrClient, nodesClient, db)
+	srv.schedulerServer = scheduler.New(scanner, workflowManager)
```

Comment:
If we move some code around in startup, I think we can further simplify this by passing in the scheduler directly.